build/networking/README_DOCKER.md

```diff
@@ -130,7 +130,7 @@ configuration file settings and inner workings!
     mkdir -p "${VOLUME}"/intf
   * docker run --name "${CONTAINER_NAME}" --cap-add ALL --privileged \
                -v "${VOLUME}":/tmp -p 9339:9339 -p 9559:9559 \
-               -d --entrypoint /root/scripts/start.sh -it <Image ID> rundaemon
+               -d --entrypoint /root/scripts/start.sh -it < Image ID > rundaemon
 
 * Run `ipdk connect` - To connect to your IPDK container daemon and to use
   it from a command line. Everytime we login to container via `ipdk connect`,
```

Comment:
"Every time we log in to the container ..."

_Every time_ is two words
_log in_ is two words
_container_ takes an article

Reply:
Old text from other authors...I just fixed the formatting :)

Fixed